src/proof/lean/lean_post_processor.cpp

```diff
@@ -1304,6 +1305,75 @@ bool LeanProofPostprocessCallback::update(Node res,
                   *cdp);
       break;
     }
+    case PfRule::ARITH_MULT_SIGN:
+    {
+      std::vector<Node> variables;
+      std::vector<Node> pols;
+      size_t numArgs = args.size();
+      // last element is the conclusion
+      for (size_t i = 0; i < numArgs - 1; i++)
+      {
+        Assert(arg[i].getKind() == Kind::NOT ||
+               arg[i].getKind() == Kind::LT  ||
+               arg[i].getKind() == Kind::GT);
+        const Node& currArg = args[i].getKind() == Kind::NOT ? args[i][0] : args[i];
+        Node toPush;
+        int32_t pol;
+        Assert(!currArg[0].isConst()
+               || currArg[0].getConst<Rational>() == Rational(0));
+        Assert(!currArg[1].isConst()
+               || currArg[1].getConst<Rational>() == Rational(0));
+        if (currArg[0].isConst())
+        {
+          toPush = currArg[1];
+          pol = currArg.getKind() == Kind::LT      ? 1
```

Comment:
clang-format

Reply:
hmm, this was the output of clang-format for me... can you share the specific version and which specific flag do you set?

Reply:
You are right. I wrongly assumed this was not clang-formatted hehe